app/Http/Controllers/Api/V1/SuborganizationController.php

```diff
@@ -880,4 +882,69 @@ public function userHasPermission(SuborganizationUserHasPermissionRequest $subor
             'userHasPermission' => auth()->user()->hasPermissionTo($data['permission'], $suborganization),
         ], 200);
     }
+
+    /**
+     * Get Organization Media Sources
+     *
```

Comment:
add params in doc block (e.g. Organization $organization)

Reply:
added